src/state/operator/selectors.js

```diff
@@ -92,27 +143,18 @@ export const selectSocketIdentity = ( { operators: { sockets, identities } }, so
 	get( sockets, socket.id )
 )
 export const selectUser = ( { operators: { identities } }, userId ) => get( identities, userId )
-export const selectTotalCapacity = ( locale, groups, state ) => compose(
-	reduce( ( { load: totalLoad, capacity: totalCapacity }, { id, status, online } ) =>
-		ifElse(
-			whereEq( { status: STATUS_AVAILABLE, online: true } ),
-			() => {
-				const { load, capacity, active } = getLocaleMembership( locale, id, state )
-				if ( ! active || ! isMemberOfGroups( id, groups ) ) {
```

Comment:
Deleting this would not check if the operator has the locale active or is part of the requested locale group. Is this intentional? I can't say that I'm really sure how this should work :)